test/io.jl

```diff
@@ -490,4 +491,13 @@ module TestIO
     # Need to wrap macro call inside eval to prevent the error from being
     # thrown prematurely
     @test_throws ArgumentError eval(:(csv"foo,bar"a))
+
+    # Test LaTeX export
+    df = DataFrame(A = 1:4, 
+                   B = ["\$10.0", "M&F", "A~B", "\\alpha"], 
+                   C = [L"\alpha", L"\beta", L"\gamma", L"\sum_{i=1}^n \delta_i"],
+                   D = [1.0, 2.0, Nullable(), 3.0]
+                   )
+    @test isequal(reprmime(MIME("text/latex"), df), "\\begin{tabular}{r|cccc}\n\t& A & B & C & D\\\\ \n\t\\hline \n\t1 & 1 & \\\$10.0 & \$\\alpha\$ & 1.0 \\\\ \n\t2 & 2 & M\\&F & \$\\beta\$ & 2.0 \\\\ \n\t3 & 3 & A\\textasciitilde{}B & \$\\gamma\$ &  \\\\ \n\t4 & 4 & \\textbackslash{}alpha & \$\\sum_{i=1}^n \\delta_i\$ & 3.0 \\\\ \n\\end{tabular}\n")
```

Comment:
I think using a multiline string block (`"""`) you should be able to replace the `\n` by actual line breaks to make it more readable.